### PR TITLE
Data Privacy Day survey + email

### PR DESCRIPTION
**Pull request by TheoChevalier**

### CODE REVIEW

**Comment by TheoChevalier** [comment]
For the survey, question 11 and “Other - Please offer your own” will be hidden on localized versions, so not exposing those strings.

File: en-US/surveys/data_privacy_day.lang [hunk 1]
```diff
@@ -0,0 +1,357 @@
...
+
+
+# q-5
+;My knowledge of how to protect myself online is best described as:
+My knowledge of how to protect myself online is best described as:
+
+
+# q-5-o-10003
+;I’m all over it. I know all about encryption, 2Fa, all of it. I’m basically Mr. Robot.
+I’m all over it. I know all about encryption, 2Fa, all of it. I’m basically Mr. Robot.
```
Comment (flodolo) [inline en-US/surveys/data_privacy_day.lang]: Not that it matters, but I always see this written as 2FA

File: en-US/surveys/data_privacy_day.lang [hunk 2]
```diff
@@ -0,0 +1,357 @@
...
+Which best describes your knowledge of encryption:
+
+
+# q-6-o-10006
+;I use it all the time. Love it. I’m basically Alan Turing.
+I use it all the time. Love it. I’m basically Alan Turing.
+
+
+# q-6-o-10007
+;I know a little about encryption.  I wish I knew more.
```
Comment (flodolo) [inline en-US/surveys/data_privacy_day.lang]: English double space after period

File: en-US/surveys/data_privacy_day.lang [hunk 3]
```diff
@@ -0,0 +1,357 @@
...
+Sometimes — I do my best, but there is only so much I can do
+
+
+# q-7-o-10012
+;Rarely — Why bother trying to protect my personal information when Google and Facebook already know everything there is to know about me?
+Rarely — Why bother trying to protect my personal information when Google and Facebook already know everything there is to know about me?
+
+
+# q-7-o-10013
+;Never —  I feel like Big Brother is watching me and I have no privacy
```
Comment (flodolo) [inline en-US/surveys/data_privacy_day.lang]: English double space after dash

File: en-US/surveys/data_privacy_day.lang [hunk 4]
```diff
@@ -0,0 +1,357 @@
...
+I’m willing to spend all the time I need to make myself safe and secure online
+
+
+# q-20-o-10050
+;I don’t have any time for this
+I don’t have any time for this
+
+
+# q-21
+;The number of devices I own that connect to the internet (including computers, phones, fitness trackers, internet connected cars, thermostats, or appliances, wifi routers, TV/cable boxes, etc).
```
Comment (flodolo) [inline en-US/surveys/data_privacy_day.lang]: Wi-Fi

**Pull request merged**